### DISCUSSION TITLE
Moving ECM in Task 1

### DISCUSSION THREAD

**Discussion by jiang-yiwei** [surgical-robotics-ai/surgical_robotics_challenge#46] [Q&A] [answered]
The challenge allows us to move the ECM, so let's say, we move it from T_ecm1 to T_ecm2 to get a different view of the needle, but we only need to output the needle pose wrt. to the first ECM pose, i.e. T_ecm1_to_needle, is that right?

**Comment by pkazanzides** [comment]
We are still discussing this, but I believe the best option is to report the needle pose with respect to the current ECM pose.

**Comment by pkazanzides** [comment, marked as answer]
We clarified the challenge website to indicate that the needle pose should be reported with respect to the current ECM pose (i.e., the pose of the ECM when the completion report is submitted via either ROS topic or Python method call). Note that we also increased the time limit from 10 seconds to 60 seconds.
